receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper.go

```diff
@@ -436,3 +448,17 @@ func (s *scraper) scrapeAndAppendSignalsPendingMetric(ctx context.Context, now p
 
 	return nil
 }
+
+func (s *scraper) scrapeAndAppendSystemProcessesCreated(ctx context.Context, now pcommon.Timestamp, data aggregateProcessMetadata) {
+	if s.config.MetricsBuilderConfig.Metrics.SystemProcessesCreated.Enabled {
+		s.mb.RecordSystemProcessesCreatedDataPoint(now, data.processesCreated)
+	}
+}
+
+func (s *scraper) scrapeAndAppendSystemProcessesCount(ctx context.Context, now pcommon.Timestamp, data aggregateProcessMetadata) {
+	if s.config.MetricsBuilderConfig.Metrics.SystemProcessesCount.Enabled {
```

Comment:
This check isn't needed here because `RecordSystemProcessesCountDataPoint` already has it, but it would be nice to wrap `s.getAggregateProcessMetadata(phs)` with it so we don't throw errors for disabled metrics

Reply:
There is a check in `s.getAggregateProcessMetadata` that checks first for either of the `system.process.*` metric to be enabled. Is that what you were intending, or is there another way we should guard it?

Reply:
Ok, we're good then